### PR TITLE
Add gradle files for typescript containers and update the version info.

### PR DESCRIPTION
**Pull request by robertwb**
Container are now built with
    
    ./gradlew ":sdks:typescript:container:docker"
    
just like all other SDKs and they will be pushed as part of the release process.

Also fixes issue with typescript sibling sdks.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by robertwb** [comment]
R: @damccorm

File: sdks/typescript/container/boot.go [hunk 1]
```diff
@@ -169,13 +169,14 @@ func main() {
...
 
-  workerIds := append([]string{*workerId}, info.GetSiblingWorkerIds()...)
+  workerIds := append([]string{*id}, info.GetSiblingWorkerIds()...)
 	var wg sync.WaitGroup
 	wg.Add(len(workerIds))
 	for _, workerId := range workerIds {
 		go func(workerId string) {
-			log.Printf("Executing: python %v", strings.Join(args, " "))
-			log.Fatalf("User program exited: %v", execx.ExecuteEnv(map[string]string{"WORKER_ID": workerId}, "npx", args...))
+		  workerArgs := append(append([]string{}, args...), "--id=" + workerId)
```
Comment (damccorm) [inline sdks/typescript/container/boot.go:177]: Looks like this file might need a `go fmt`

File: sdks/typescript/build.gradle [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          ])
+          )
+
+def pack = tasks.register("pack") {
+
+  doLast {
+    // Build artifact
+    exec {
+      executable 'sh'
+      args '-c', "npm run build && npm pack --pack-destination ${buildDir}"
```
[the remark this answers is not in the document]
Comment (robertwb) [reply sdks/typescript/build.gradle]: Good call. Done.

File: sdks/typescript/container/boot.go (hunk 1 above)
Comment (robertwb) [reply sdks/typescript/container/boot.go]: Done.

**Review by damccorm — APPROVED** [review]

**Pull request merged**